consul_kv_migrator.py

```diff
@@ -0,0 +1,97 @@
+from consul_kv import Connection
+import json
+import ast
+from prettytable import PrettyTable
+import getopt
+import sys
+
+argv = sys.argv[1:]
```

Comment:
missing main function .. Please add one create one function for parsing user inputs